src/core/Akka/Pattern/BackoffOnRestartSupervisor.cs

```diff
@@ -26,7 +26,8 @@ internal sealed class BackoffOnRestartSupervisor : BackoffSupervisorBase
             TimeSpan maxBackoff,
             IBackoffReset reset,
             double randomFactor,
-            OneForOneStrategy strategy) : base(childProps, childName, reset)
+            OneForOneStrategy strategy,
+            object replyWhileStopped) : base(childProps, childName, reset, replyWhileStopped)
```

Comment:
Same case as with BackoffSupervisor constructor - potential break of API contract.